lib/bearerstrategy.js

```diff
@@ -34,6 +36,13 @@ const jws = require('jws');
  * signature is:
  *
  *     function(token, done) { ... }
+ * or
+ *     function(req, token, done) { ... }
```

Comment:
This is the pattern of passport js, and req is not an "optional" argument. In order to use the signature with req, user has to explicitly set passReqToCallback true in the options variable when they do Passport.use(new strategy(options, verify)). Passport strategy will read the passReqToCallback value to decide if this verify callback's first parameter is req. I mentioned this between line 42-45 and 72.